Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -939,5 +940,21 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             Message = string.Join(": ", crawlerName, Loc.GetString("admin-smite-crawler-description"))
         };
         args.Verbs.Add(crawler);
+
+        var scurretName = Loc.GetString("admin-smite-scurret-species-swap-name").ToLowerInvariant(); // Started with a CNTL+C CNTL+V of lizard smite
+        Verb scurret = new() // this is my first attempt at adding a smite. it just worked when i tested it. why did nothing break. i dont expect things working first try
+        {
+            Text = scurretName,
+            Category = VerbCategory.Smite,
+            Icon = new SpriteSpecifier.Rsi(new("/Textures/Mobs/Animals/scurret/scurret.rsi"), "scurret_oof"), //gotta get a scurret icon for this
+            Act = () =>
+            {
+                _polymorphSystem.PolymorphEntity(args.Target, "AdminScurretSmite");
```

Comment:
This identifier needs to be put into a `static readonly ProtoId<PolymorphPrototype>`.